halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -173,6 +174,9 @@ private void safeExecuteThis() {
       AnsiUi.error(e.getMessage());
       AnsiUi.remediation("Try the command again with the --debug flag.");
       System.exit(1);
+    } catch (DaemonTaskInterrupted e) {
+      AnsiUi.warning("Interrupted. Task killed in unknown state.");
```

Comment:
This happens when a different user kills your task. Since that can happen at anytime, there is no good way to know what just happened (beyond looking at the task output on your screen).